example/cmd/data-restorer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine as builder
+RUN apk update && apk add git gcc libc-dev && rm -rf /var/cache/apk/*
+ARG GITHUB_TOKEN
```

Comment:
Do we have any private dependency that needs this?
Also, it's not in the scope of the PR, but could you take a look at https://github.com/qor5/admin/blame/84962acce755363770fb247dd3d7de70024be66b/example/cmd/publisher/Dockerfile#L3 too?

Reply:
No private dependency used here, I'll remove github token.(also in the other file)